infra-templates/k8s/32/rancher-compose.yml

```diff
@@ -29,11 +29,23 @@
     default: false
     type: boolean
   - variable: REGISTRY
-    description: The registry to pull addon images and the optional pod infra container image. This should be set to the private registry from which the images for addon-services (i.e. helm, heapster, dashboard, dns) and the pod infra container image should be pulled.
+    description: The registry to pull addon images and the optional pod infra container image. This should be set to the private registry from which the images for addon-services (i.e. helm, heapster, dashboard, dns) and the pod infra container image should be pulled ( Default：gcr.io ).
     label: Private Registry for Add-Ons and Pod Infra Container Image
     required: false
     default: ""
     type: string
+  - variable: IMAGE_NAMESPACE
+    description: images namespace for Add-Ons and Pod Infra Container Image ( Default：google_containers ).
+    label: Image namespace for  Add-Ons and Pod Infra Container Image
+    required: false
+    default: ""
+    type: string
+  - variable: KUBERNETES_HELM
```

Comment:
as both parameters are namespaces, could we be consistent in naming things? Like HELM_IMAGE_NAMESPACE, BASE_IMAGE_NAMESPACE? Up to suggestions, but the names should reflect the meaning

Reply:
@hongxiaolu Let's do `HELM_IMAGE_NAMESPACE` and `BASE_IMAGE_NAMESPACE` like Alena suggested.